processing/manual_correction.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Run this script in results/data folder
+#
+# Manual segmentations and labels are saved into ../../seg_manual folder
+# (feel free to change this folder by change PATH_SEGMANUAL variable)
+#
+# USAGE:
+# Define files.yml file as following:
+#   FILES_SEG:
```

Comment:
we have not distinguished GM from SC seg... we probably should

Reply:
Was done based on t2star contrast:

https://github.com/spine-generic/spine-generic/blob/957bf91d14250c26527f2b9debea821824fbad8d/processing/manual_correction.sh#L79

Do you think that separate keys `FILE_SEG_SC` and `FILE_SEG_GM`, how is proposed here are better?:
 https://github.com/spine-generic/spine-generic/blob/3b7fbeccc3e5b891b5271b896bb70d557ba471cb/processing/manual_correction.py#L11

For me, it is more understandable at the first look because user explicitly defines SC or GM.

Reply:
> Was done based on t2star contrast:

ok all good then